src/DataCollector/MongoQuerySerializer.php

```diff
@@ -63,11 +67,10 @@ public static function prepareItemData($item)
             return $item->bsonSerialize();
         }
 
-        if (is_array($item) || is_object($item)) {
+        if (\is_array($item) || \is_object($item)) {
```

Comment:
Because the Inspection plugin suggested it :smile: it's an optimization, see https://veewee.github.io/blog/optimizing-php-performance-by-fq-function-calls/

Reply:
I know that is an optimization but in dev env I do not care about this gain and furthermore you are preventing moking of these function for testing

Reply:
Reverting :+1: